target/linux/ipq806x/patches-5.4/0069-arm-boot-add-dts-files.patch

```diff
@@ -29,9 +29,11 @@ Signed-off-by: John Crispin <john@phrozen.org>
 +	qcom-ipq8064-wg2600hp.dtb \
 +	qcom-ipq8064-wpq864.dtb \
 +	qcom-ipq8064-wxr-2533dhp.dtb \
++	qcom-ipq8065-rt4230w-rev6.dtb \
 +	qcom-ipq8065-nbg6817.dtb \
 +	qcom-ipq8065-r7800.dtb \
 +	qcom-ipq8068-ecw5410.dtb \
  	qcom-msm8660-surf.dtb \
  	qcom-msm8960-cdp.dtb \
  	qcom-msm8974-fairphone-fp2.dtb \
+
```

Comment:
Remove empty line here.

Reply:
Sorry about all these weird things. gedit just likes to mess with things for some reason.